lerna.json

```diff
@@ -4,13 +4,17 @@
     "packages/*"
   ],
   "version": "1.10.0-rc.0",
+  "useWorkspaces": true,
```

Comment:
more information on workspaces, see https://blog.kintone.io/entry/2020/07/14/103322. Other companies who use workspaces are vercel/next (which have a similar problem to us with React examples + packages)